avionics/tests/unit_States.cpp

```diff
@@ -37,6 +48,89 @@ TEST(AscentToApogee, FireIgnitor) {
     EXPECT_TRUE(-1 < t && t < 1);
 }
 
+TEST(AscentToApogee, MACH_LOCK) {
```

Comment:
Googletest naming doesn't allow underscores in test names; UpperCamelCase is what other tests uses

Reply:
(Also applies to the other tests)